resources/carrier/en/234.txt

```diff
@@ -151,7 +155,13 @@
 2347028|Starcomms
 2347029|Starcomms
 234703|MTN
-234704|Visafone
+2347040|Visafone
```

Comment:
As complete 704 range is assigned to Visafone as per ITU doc. https://www.itu.int/dms_pub/itu-t/oth/02/02/T020200009C0001PDFE.pdf, we need not to be elaborative when mentioning carrier.